Dialog/RoomPreviewGraphicsView.h

```diff
@@ -16,7 +16,7 @@ class RoomPreviewGraphicsView : public QGraphicsView
 
 private:
     bool initialized = false; // Used for initializing the scrollbars
-    void *dataPointers[3]; // Tilset, BG, L0
+    void *dataPointers[4] = {0, 0, 0, 0}; // Tilset, BG, L0
```

Comment:
Why are there 4? Also, this is already initialized at RoomPreviewGraphicsView.cpp:19